pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -1283,30 +1303,41 @@ func (b *MacvtapBindMechanism) decorateConfig() error {
 	return nil
 }
 
-func (b *MacvtapBindMechanism) loadCachedInterface(pid, name string) (bool, error) {
-	ifaceConfig, err := b.storeFactory.CacheForPID(pid).Read(name)
+func (b *MacvtapBindMechanism) getPIDString() string {
```

Comment:
We have already a getPIDString free function https://github.com/kubevirt/kubevirt/pull/5059/files#diff-030aa8b7ff9bf4a511d1d5641c1dec04205a944137672be3a691f0b1b4068753R548

Reply:
Oh. Right. Gone.